Forms/MainForm.cs

```diff
@@ -75,12 +78,22 @@ private void UpdateConfigurationSubstitutions()
         {
             Configuration.DefaultSubstitutions.Clear();
             Configuration.DefaultSubstitutions.AddRange(substitutionList1.Substitutions);
+            LoadConfigurationDetails();
         }
 
 
         private void LoadConfigurationDetails()
         {
-            txtWordList.Text = Configuration.DefaultWords.ToString();
+            string wordtext = Configuration.DefaultWords.ToString();
+            if (txtWordList.Text != null && txtWordList.Text != wordtext && txtWordList.Text != wordtext + " ")

```

Comment:
Oh, I see what you're saying. It might work. This line of code was to fix a
problem I was having, where the cursor would jump, and the trailing space
get removed, whenever I added a word; because the text box was over-written
with the split+joined content that had any empty trailing item removed.

It's not a pretty workaround but it solved the problem. I'd certainly
prefer a better. Not having focus might work as well and be more
elegant...but doesn't the word list entry have focus on first load? Will
that prevent the initial content from loading?
On Nov 15, 2015 2:58 PM, "Pascal" <notifications@github.com> wrote:

> In Forms/MainForm.cs
> <https://github.com/fireout/keepasssequencer/pull/17#discussion_r44877439>
> :
>
> >          }
> >
> >
> >          private void LoadConfigurationDetails()
> >          {
> > -            txtWordList.Text = Configuration.DefaultWords.ToString();
> > +            string wordtext = Configuration.DefaultWords.ToString();
> > +            if (txtWordList.Text != null && txtWordList.Text != wordtext && txtWordList.Text != wordtext + " ")
>
> But if the wordList has the focus, any changes are directly applied to
> Configuration.DefaultWords? we dont need to update the input with the
> same value? The strength bar can still be updated?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/fireout/keepasssequencer/pull/17/files#r44877439>.
>